src/support_tests.cc

```diff
@@ -179,15 +190,17 @@ bool test_authenticated_encrypt(bool print_all) {
     printf("\n");
   }
   if (size_decrypt_out != size_in || memcmp(plain, decrypted, size_in) != 0) {
-    printf("comparison failed\n");
+    printf("%s() error, line: %d, comparison failed\n",
+         __func__, __LINE__);
     return false;
   }
 
   size_encrypt_out = out_size;
   size_decrypt_out = out_size;
 
   if (!authenticated_encrypt("aes-256-gcm", plain, in_size, key, iv, cipher, &size_encrypt_out)) {
-    printf("authenticated encrypt for aes-256-gcm failed\n");
+    printf("%s() error, line: %d, authenticated for aes-256-gcm encrypt failed\n",
```

Comment:
I thought the phrasing in the old message was better.

Rearrange for improved readability to: `authenticated encrypt  for aes-256-gcm failed`

Reply:
done